### ISSUE TITLE
DataPack problem

### ISSUE THREAD

**Issue by Gamingworldhun** [ObliviousSpartan/SpartanWeaponry-API#13]
I am using 1.18.2 version of minecraft with beta 1 of spartan weaponry forge version is 40.2.14 when i try to load a world with the weaponry mod i got a message with a problem with the datapack and only with the weaponry shields work fine it doesn't cause this only the weaponry

**Comment by ObliviousSpartan** [comment]
Hi there,
That is weird. There shouldn't be any problems with the mod's datapack.
Can you please provide me the log file when the mod was launched? It should provide info that will help me zero in on what is going wrong. You should be able to find it in the "logs" folder in your modded Minecraft instance.

**Comment by Gamingworldhun** [comment]
okay I share the logs but what I also assume there is a incompatibilyti (sry for my bad english) with a other mod(s) that i use
[2023-12-26-7.log.gz](https://github.com/ObliviousSpartan/SpartanWeaponry/files/13779977/2023-12-26-7.log.gz)

**Comment by ObliviousSpartan** [comment]
The log you provided me is one of the logs which didn't have Spartan Weaponry loaded at all. It did provide me some mods to look into though.
After looking into it and loading some mods into a test instance. I found out that it is a cross-mod incompatibility with Farmers Delight of all things!
It has something to do with Farmers Delight looking up Tool Actions for its ingredients for some reason
[FarmersDelightXSpartanWeaponryCrash.log](https://github.com/ObliviousSpartan/SpartanWeaponry/files/13782841/FarmersDelightXSpartanWeaponryCrash.log)
I also just found this on their GitHub with the exact same issue: https://github.com/vectorwing/FarmersDelight/issues/857
I just need to add some sanity checks to make sure that it doesn't cause a datapack problem anymore.
